storybook/src/example-pages/toepassingen/regels-op-de-kaart/documenten-VDK.ts

```diff
@@ -0,0 +1,159 @@
+          <div class="dso-document-header">
+            <div class="dso-highlight-box">
+              ${contextTemplate({
+                type: "label",
+                label: html`<h2>Omgevingsplan gemeente Apeldoorn</h2>`,
+                content: html`${iconTemplate({ icon: "map-location" })}`,
+                children: html` <p>Omgevingsplan - Gemeente Apeldoorn</p>
+                  ${buttonTemplate({
+                    label: "meer kenmerken",
+                    type: "button",
+                    variant: "tertiary",
+                    iconMode: "after",
+                    compact: true,
+                    icon: { icon: "chevron-down" },
+                  })}
+                  ${advancedSelectTemplate({
+                    active: { label: "In werking - Laatst gewijzigd: 01-01-2024" },
+                    options: [
+                      {
+                        label: "Geldende versie",
+                        variant: "success",
+                        redirect: {
+                          href: "#",
+                          label: "Bekijk eerder vastgestelde versies",
+                        },
+                        options: [
+                          {
+                            label: "In werking (laatst gewijzigd: 01-01-2024)",
+                          },
+                        ],
+                      },
+                    ],
+                  })}`,
+              })}
+            </div>
+          </div>
+          ${navbarTemplate(documentPanelSubmenu)}
+          <div class="dso-highlight-box">
+            <div class="dso-button-row dso-align-right">
+              ${selectableTemplate({
+                type: "checkbox",
+                id: "waarde1",
+                value: "compacte weergave",
+                label: "Compact weergave",
+                name: "weergave",
+              })}
+            </div>
+          </div>
+          ${accordionTemplate({
+            variant: "compact-black",
+            reverseAlign: true,
+            sections: [
+              {
+                handleTitle: "Activiteiten(90)",
+                heading: "h4",
+                open: true,
+                content: html`
+                  ${plekinfoCardsListActiviteiten.map((plekinfoCard) =>
+                    plekinfoCardTemplate({
+                      ...plekinfoCard,
+                      symbool: html`${iconTemplate({ icon: "house" })}`,
+                    }),
+                  )}
+                  <a href="#">Toon alle activiteiten</a>
+                `,
+              },
+
+              {
+                handleTitle: "Locaties (3)",
+                heading: "h4",
+                open: true,
+                content: html`
+                  ${plekinfoCardsListLocaties.map((plekinfoCard) =>
+                    plekinfoCardTemplate({
+                      ...plekinfoCard,
+                      symbool: html`${iconTemplate({ icon: "house" })}`,
```

Comment:
Ik weet het niet zeker, maar misschien mag:

```suggestion
                      symbool: iconTemplate({ icon: "house" }),
```

Reply:
Yes, dat werkt

Reply:
Ok, maar waarom is het dan niet verwerkt?

edit: Volgens mij heb je het alleen op regel 196 doorgevoerd

Reply:
klopt, zag em net nog voorbij komen. Excuus